scripts/ufm_fabric_health/README.md

```diff
@@ -0,0 +1,83 @@
+UFM Fabric Health
+--------------------------------------------------------
+
+
+This script is used to run Fabric Health Report via the UFM APIs.
+
+
+Prerequisites
+--------------------------------------------------------
+
+To install all prerequisites, please run :
+
+    pip install -r requirements.txt
+
+Run
+--------------------------------------------------------
+### 1. Using command line arguments :
+
+    python3 run_fabric_health_report.py --cables=True
+
+
+
+### 2. Using configuration file :
+    
+  - Copy main SDK config file sample ufm-sdk.sample.cfg to ufm-sdk.cfg
+
+
+        cp ../conf/ufm-sdk.sample.cfg ../conf/ufm-sdk.cfg
+
+  - Edit config file with relevant parameters
+
+
+        vi ../conf/ufm-sdk.cfg
+    
+  - Copy ufm fabric health config file sample fabric_health.sample.cfg to fabric_health.cfg
+
+
+        cp fabric_health.sample.cfg fabric_health.cfg
+
+  - Edit config file with relevant parameters
+
+
+        vi fabric_health.cfg
+
+  - Run
+
+
+        python3 run_fabric_health_report.py
+
+
+ Running syntax
+--------------------------------------------------------
+
+| Argument | Corresponding Config Value | Required | Description |
+| :---: | :---: |:---: |:---: |
+| --ufm_host <ufm_host> | [ufm-remo `te-server-config.host](../conf/ufm-sdk.sample.cfg#L2) | True | Hostname or IP for The UFM Enterprise
+| --ufm_protocol <ufm_protocol> | [ufm-server-config.ws_protocol](../conf/ufm-sdk.sample.cfg#L4) | True | Web services protocol used by UFM Enterprise (HTTP, HTTPS)
+| --ufm_username <ufm_username> | [ufm-server-config.username](../conf/ufm-sdk.sample.cfg#L6) | True | Username of UFM user
+| --ufm_password <ufm_password> | [ufm-server-config.password](../conf/ufm-sdk.sample.cfg#L7) | True | Password of UFM user
+| --logs_file_name <logs_file_name> | [logs-config.logs_file_name](../conf/ufm-sdk.sample.cfg#L11) | False | Log file name [Default = 'console.log']
+| --logs_level <logs_level> | [logs-config.logs_level](../conf/ufm-sdk.sample.cfg#L14) | False | Default is 'info'
+| --duplicate_nodes <duplicate_nodes> | [fabric-health.duplicate_nodes](fabric_health.sample.cfg#L3) | True | Duplicated Node Description
```

Comment:
all report arguments should have enough description,
please review and add a brief description using the below table:
https://docs.nvidia.com/networking/display/UFMEnterpriseUMv69/Fabric+Health+Tab